docs/getting-started.md

```diff
@@ -50,6 +50,35 @@ checked-in [deployment yaml](/build/yamls/antrea.yml):
 kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
 ```
 
+### Deploying Antrea on a Cluster with Existing CNI
+
+Ideally Antrea should only be deployed on a new cluster, which is not already
```

Comment:
Maybe just say: the above description is only for deploying Antrea in a new cluster.

Reply:
Done